content/docs/concepts/traffic-management/routing/_index.md

```diff
@@ -0,0 +1,151 @@
+
+You might use all or only some of the Istio network resources, depending on
+your platform and your use case. Your platform handles basic traffic routing,
+but configurations for advanced use cases might require the full range of Istio
+traffic routing features.
+
+### Routing traffic to multiple versions of a service {#routing-versions}
+
+Typically, requests sent to services use a service's hostname or IP address,
+and clients sending requests don't distinguish between different versions of
+the service.
+
+With Istio, because the Envoy proxy intercepts and forwards all requests and
+responses between the clients and the services, you can use routing rules with
+service subsets in a virtual service to configure the traffic routes for
+multiple versions of a service.
+
+The following diagram shows a configuration that relies on routing rules to
+handle the communication between a client and a service that has multiple
+versions.
+
+Envoy proxies dynamically determine where to send the traffic based on the
+routing rules you configure in the [ingress gateway](/docs/concepts/traffic-management/routing/gateways/)
+and [virtual service](/docs/concepts/traffic-management/routing/virtual-services).
+In this example, those rules route the incoming request to v1, v2, or v3 of
+your application's service.
+
+You use the Istio [networking APIs](/docs/reference/config/networking/)
+to configure [network resources](/docs/concepts/traffic-management/routing/)
+and specify the [routing rules](/docs/concepts/traffic-management/routing/virtual-services/#routing-rules).
+
+The advantage of this configuration method is that it decouples the application
+code from the evolution of its dependent services. This in turn provides
+monitoring benefits. For details, see [Mixer policies and telemetry](/docs/concepts/policies-and-telemetry/).
+
+### Canary rollouts with autoscaling {#canary}
+
+Canary rollouts allow you to test a new version of a service by sending a small
+amount of traffic to the new version. If the test is successful, you can
+gradually increase the percentage of traffic sent to the new version until all
+the traffic is moved. If anything goes wrong along the way, you can abort the
+rollout and return the traffic to the old version.
+
+Container orchestration platforms like Docker, or Kubernetes support canary
+rollouts, but they use instance scaling to manage traffic distribution, which
+quickly becomes complex, especially in a production environment that requires
+autoscaling.
+
+With Istio, you can configure traffic routing and instance deployment as
+independent functions. The number of instances implementing the services can
+scale up and down based on traffic load without referring to version traffic
+routing at all. This makes managing a canary version that includes autoscaling
+a much simpler problem. For details, see the [Canary Deployments](/blog/2017/0.1-canary/)
+blog post.
+
+### Splitting traffic for A/B testing {#splitting}
```

Comment:
This section is not coherent with the above. Canary and A/B are related. As far as Istio is concerned, these are simply marketing terms. We treat both as simply traffic splitting and steering. Splitting implies we split the 100% into N parts. Steering implies we look at the content of a request (like headers, browsers where its coming from, etc.) and steer 100% of that traffic to a specific subset.

Reply:
@rshriram I don't disagree with your assessment, from a technical POV. However, readers might not be aware of that nuance. I would love to collaborate on how we can better explain it in a future iteration. For now, let's keep them separate.